packages/camera/camera_windows/test/camera_windows_test.dart

```diff
@@ -5,14 +5,17 @@
 import 'package:async/async.dart';
 import 'package:camera_platform_interface/camera_platform_interface.dart';
 import 'package:camera_windows/camera_windows.dart';
+import 'package:camera_windows/src/messages.g.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter/widgets.dart';
 import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/annotations.dart';
+import 'package:mockito/mockito.dart';
 
-import './utils/method_channel_mock.dart';
```

Comment:
This "mock the channel and log the method" structure was pretty common in our older Dart unit tests. It was annoying because there was no way to do partial matches; if you wanted to assert that one argument of a 15 argument-dict method was a specific value, you had to list all 15, making it hard to see what the test was specifically about.